chromium/test/rules_test.js

```diff
@@ -96,14 +96,35 @@ describe('rules.js', function() {
   })
 
   describe('RuleSets', function() {
+    let rules_json = JSON.parse('[{"name":"Freerangekitten.com","rule":[{"to":"https:","from":"^http:"}],"target":["freerangekitten.com","www.freerangekitten.com"]}]');
```

Comment:
Perhaps better to inline the object? Result would be the same but would look slightly more readable in the source (and with proper syntax highlighting)